pkgs/applications/networking/cluster/argocd-autopilot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argocd-autopilot";
+  version = "0.2.25";
```

Comment:
Please update to the latest (https://github.com/argoproj-labs/argocd-autopilot/releases/tag/v0.2.30).